AWS: Cleanup provisioning scripts

Edwin-Pau opened the pull request:
Applied the same changes from the GCP provisioning script cleanup branch. These changes in the provisioning scripts include cleaning up of the logging messages, refactoring of various functions, and renaming the startup scripts to provisioning scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Cleanup CAC provisioning script

Applied the same changes in AWS deployment for refactoring and
cleaning up of the CAC provisioning script from the GCP deployment.

These changes include: fixing the log files, refactoring into functions,
and fixing logging messages so they are consistent across all scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Cleanup CentOS WS provisioning scripts

Applied the same changes in AWS deployment for refactoring and
cleaning up of the CentOS workstations provisioning scripts from the GCP
deployment.

These changes include: fixing the log files, refactoring into functions,
and fixing logging messages so they are consistent across all scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Cleanup Windows WS provisioning scripts

Applied the same changes in AWS deployment for refactoring and
cleaning up of the CentOS workstations provisioning scripts from the GCP
deployment.

These changes include: fixing the log files, refactoring into functions,
and fixing logging messages so they are consistent across all scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Cleanup DC provisioning scripts

Applied the same changes in AWS deployment for refactoring and
cleaning up of the CentOS workstations provisioning scripts from the GCP
deployment.

These changes include: fixing the log files, refactoring into functions,
and fixing logging messages so they are consistent across all scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Rename startup to provisioning scripts (CAC)

CAC startup scripts have been renamed to provisioning scripts and
all associated variables and names have been changed to better reflect
the functions of these scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Rename startup to provisioning scripts (WS)

CentOS-gfx, CentOS-std, Win-gfx and Win-std startup scripts have been
renamed to provisioning scripts and all associated variables and names
have been changed to better reflect the functions of these scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Rename startup to provisioning scripts (DC)

DC startup scripts have been renamed to provisioning scripts and all
associated variables and names have been changed to better reflect
the functions of these scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
AWS: Rename startup to provisioning (.sample)

The references to startup scripts in terraform.tfvars.sample files
have been changed to provisioning scripts so that the wording is
consistent with the rest of the scripts.

Signed-off-by: Edwin-Pau <epau@teradici.com>

shermanyin commented on deployments/aws/lb-connectors/terraform.tfvars.sample:
```diff
@@ -92,7 +92,7 @@ centos_std_instance_count = 0
 # plaintext passwords as strings in the variables below. Note that plaintext
 # passwords may show in Terraform logs and .tfstate files, appear unencrypted
-# in the storage bucket holding the startup scripts, as well as in the startup
+# in the storage bucket holding the provisioning scripts, as well as in the provisioning
```
`> 80 chars

shermanyin commented on deployments/aws/single-connector/terraform.tfvars.sample:
```diff
@@ -80,7 +80,7 @@ centos_std_instance_count = 0
 # plaintext passwords as strings in the variables below. Note that plaintext
 # passwords may show in Terraform logs and .tfstate files, appear unencrypted
-# in the storage bucket holding the startup scripts, as well as in the startup
+# in the storage bucket holding the provisioning scripts, as well as in the provisioning
```
`> 80 chars

shermanyin commented on modules/aws/centos-gfx/centos-gfx-provisioning.sh.tmpl:
```diff
@@ -7,42 +7,53 @@
+}
+
+check_required_vars() {
+    if [[ -z "$PCOIP_REGISTRATION_CODE" ]]; then
```
heads up that pcoip reg code is no longer required due to LLS support, so this will conflict with my LLS changes.
this applies to all workstations.

Edwin-Pau replied:
I will wait for LLS to merge first, then manually merge this branch with LLS changes.

shermanyin commented on modules/aws/dc/main.tf:
```diff
@@ -10,12 +10,12 @@ locals {
   # Windows computer names must be <= 15 characters
   host_name                  = substr("${local.prefix}vm-dc", 0, 15)
-  setup_file                 = "C:/Temp/setup.ps1"
+  provisioning_file                 = "C:/Temp/provisioning.ps1"
```
spacing

Commit:
Fix max characters per line in .sample files

Fixed the maximum characters per comment line in .sample files to be
80 characters or under.

Signed-off-by: Edwin-Pau <epau@teradici.com>

Commit:
Fix spacing

Fixed spacing of one of the variables in the DC main.tf file.

Signed-off-by: Edwin-Pau <epau@teradici.com>

shermanyin committed:
Merge branch 'staging' into fix/aws/provisioning-scripts
